packages/stream_chat_flutter/lib/src/message_input.dart

```diff
@@ -472,7 +461,34 @@ class MessageInputState extends State<MessageInput> {
         child: child,
       );
     }
-    return child;
+
+    return PortalEntry(
```

Comment:
I think we should rather refactor this to be one PortalEntry, and then conditionally show the correct build function in the `portal`.